TestApp/ios/Podfile.lock

```diff
@@ -249,11 +249,13 @@ PODS:
     - React
   - RNGestureHandler (1.5.0):
     - React
+  - RNScreens (2.0.0-alpha.34):
+    - React
   - Yoga (1.14.0)
 
 DEPENDENCIES:
-  - appcenter (from `../node_modules/appcenter/ios`)
   - appcenter-analytics (from `../node_modules/appcenter-analytics/ios`)
+  - appcenter-core (from `../node_modules/appcenter/ios`)
```

Comment:
appcenter is automatically changed to appcneter-core, is it correct?

Reply:
Yes